recipes/conversation_view/message_input/message_input.vue

```diff
@@ -32,14 +32,24 @@
             circle
             importance="clear"
             :aria-label="imageButtonAriaLabel"
+            @click="onSelectImage"
           >
             <template #icon>
               <dt-icon
                 name="image"
                 size="300"
               />
             </template>
+            <dt-input
+              ref="messageInputImageUpload"
+              type="file"
+              class="d-ps-absolute"
+              multiple="multiple"
```

Comment:
If I select multiple files it only fires an event for the first one.

Reply:
yea i just noticed, have no clue why, let me check..